packages/ai-ide/src/browser/app-tester-prompt-template.ts

```diff
@@ -58,7 +59,10 @@ ${REQUIRED_MCP_SERVERS.map(server => `{{prompt:mcp_${server.name}_tools}}`)}
 - **~{${IS_BROWSER_RUNNING_FUNCTION_ID}}**: Check if the browser is running. If a tool fails by saying that the connection failed, you can verify the connection by using this tool.
 - **~{${CLOSE_BROWSER_FUNCTION_ID}}**: Close the browser.
 - **~{${QUERY_DOM_FUNCTION_ID}}**: Query the DOM for specific elements and their properties. Only use when explicitly requested by the user.
-- **browser_snapshot**: Capture the current state of the page for verification or debugging purposes.
```

Comment:
this one comes from the playwright mcp server.
it is exposed in line 55 anyway but the description is different that we provide here, so maybe keep it?